front/lock.form.php

```diff
@@ -58,7 +58,31 @@
          }
       }
    }
+
+   //Execute hook to unlock fields managed by a plugin, if needed
+   Plugin::doHookFunction('unlock_fields', $_POST);
+
+} else if (isset($_POST['itemtype']) && isset($_POST["purge"])) {
+   $itemtype    = $_POST['itemtype'];
```

Comment:
This is mostly a copy/paste of the code in the other if part, should be factorized.

Also, I wonder if the `$actions` list should not be defined in `inc/define.php`.